api_types.ts

```diff
@@ -761,6 +761,12 @@ export interface SyncExecutionContext extends ExecutionContext {
   readonly sync: Sync;
 }
 
+export interface CellAutocompleteExecutionContext extends ExecutionContext {
+  readonly propertyName: string;
+  readonly propertyValues: {[propertyValues: string]: any};
```

Comment:
Nit, for the unused key alias, `otherPropertyName` or something (the dict keys are names and not values).